api/HTMLBaseElement.json

```diff
@@ -81,6 +81,39 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "forbid_data_javascript_urls": {
```

Comment:
OK, so support for forbidden urls is not symmetric between the element and its API. This shows that in current version FF and Safari support this restriction on dynamic updates https://wpt.fyi/results/html/semantics/document-metadata/the-base-element/base-data.html?label=experimental&label=master&aligned , but Chrome does not.

I know this was added in FF127. For Safari I found that the version this is supported  is 17.2 https://developer.apple.com/documentation/safari-release-notes/safari-17_2-release-notes#Resolved-Issues



https://developer.apple.com/documentation/safari-release-notes/safari-17_2-release-notes#Resolved-Issues